### ISSUE TITLE
Clade stores source files using Windows style line endings

### ISSUE THREAD

**Issue by eunovm** [17451k/clade#107]
Despite eac61dfe64245ab74448f2bcd01b3166629805e6 that seems to fix the issue, it is still there.

**Comment by eunovm** [comment]
Simple test:
```
$ file path/to/file/in/storage
.. with CRLF line terminators
```

**Comment by 17451k** [comment]
eac61df fixes internal Clade problem that is not relevant to this issue. And technically it is not a bug since nobody asked before to convert CRLF line endings to LF.

**Comment by 17451k** [comment]
Implemented in fd99a97a288aeee35f517c35a1a47a13eed69c7e.
